src/__tests__/__fixtures__/explicit-no-ssr/output.js

```diff
@@ -0,0 +1,6 @@
+import { lazyForPaint } from 'react-loosely-lazy';
```

Comment:
Did you hard code this file? Can we set the babel plugin tester to snapshot mode?

Reply:
I just copy pasted from my astexplorer stuff, didn't know we could use a snapshot thing hmm